src/lib/components/setlist/EditLineupModal.svelte

```diff
@@ -36,15 +36,18 @@
 		(user) => ({ id: user.id, text: user.name } as DropdownItem)
 	)
 
-	export let participants: {
+	export let lineup: {
 		participantId: string | undefined
 		instrument: string | undefined
 	}[]
 
-	let modalParticipants: {
-		participantId: string | undefined
-		instrument: string | undefined
-	}[] = [...participants, { participantId: undefined, instrument: undefined }]
+	// These must be reactive, so that the modal updates when the song or participants change
+	$: bandcoachId = lineup.find((p) => p.instrument?.toLowerCase() === 'bandcoach')?.participantId
```

Comment:
Are you sure you want to restrict yourself to one bandcoach?

Reply:
(added multiple bandcoach support)